bot/cogs/weather.py

```diff
@@ -0,0 +1,54 @@
+import discord
+from discord.ext import commands
+import json
+import urllib.request
+
+
+
+
+class WeatherCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
```

Comment:
missing brief and description

Reply:
Should have this one done by end of today